extension/src/test/suite/experiments/index.test.ts

```diff
@@ -897,13 +897,12 @@ suite('Experiments Test Suite', () => {
       })
 
       await inputEvent
-      await mockMessageReceived
```

Comment:
This line is not needed since `getMessageReceivedEmitter` does not return a promise.